roles/clevis/defaults/main.yml

```diff
@@ -4,4 +4,5 @@ clevis_tang_server: 127.0.0.1
 clevis_tang_server_protocol: 80
 clevis_tang_url: "http://{{ clevis_tang_server }}:{{ clevis_tang_server_protocol }}"
 clevis_initial_luks_password: password
+clevis_luks_slot: 1
```

Comment:
Should be ``2`` to remain consistent with previous role implementation, and ``1`` as overwrite in the test

Reply:
Setting the variable clevis_luks_slot to 2 is also compatible with our test suite.